index.html

```diff
@@ -153,17 +165,20 @@ <h3 class="product-manager__section-title, product-manager-title-shift">Product
         </div>
 
         <div class="product-manager__section-text-wrapper">
-          <div class="product-manager__section-icon"></div>
+          <div class="product-manager__section-icon">
+          </div>
           <p class="product-manager__section-text">We track the markets daily and know where the savings are.</p>
         </div>
       </div>
     </section>
+  </div>
 
+  <div class="marketers__section-container">
     <section class="marketers__section">
       <div class="marketers__section-description">
         <p class="marketers__section-text">Optimization for</p>
```

Comment:
<h2>
<span class="marketers__section-text">Optimization for</span>
Marketers
</h2>
all the text is header by meaning and part of it styled differently